Assignment-4/Assignment-4.cpp

```diff
@@ -264,6 +218,7 @@ bool SSE::assertchecking(const ICFGNode* inode){
         return false;
     }
     else {
```

Comment:
This assert will be triggered when students haven't added any constraints to solver.